scripts/new_project.kts

```diff
@@ -1,25 +1,68 @@
+#!/usr/bin/env kscript
+
+@file:MavenRepository("central", "https://repo.maven.apache.org/maven2/")
+@file:DependsOn("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.1")
+
+import kotlinx.coroutines.*
+import java.io.File
+import kotlin.coroutines.coroutineContext
+
 object NewProject {
 
     private const val KEY_APP_NAME = "app-name"
     private const val KEY_PACKAGE_NAME = "package-name"
     private const val ARGUMENT_DELIMITER = "="
+    private const val COROUTINE_FOLDER_NAME = "CoroutineTemplate"
```

Comment:
Should we keep the naming more general? Ex: `TEMPLATE_FOLDER_NAME`

As most likely we'll rename `CoroutineTemplate` to `template` in the future